Desktop/bioconda包推送/bioconda-recipes/recipes/gcluster/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "gcluster" %}
+{% set version = "2.0.7" %}
+{% set sha256 = "190ccf3f357d153dfb05d216a9793ce035f0512a666465bdbe5479679eda0d85" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+  
+source:
+    url: https://www.microbialgenomic.cn/gz/gcluster-v{{ version }}.tar.gz
```

Comment:
https://www.microbialgenomic.cn/gz/gcluster-v2.0.7.tar.gz does not seem to exist.

Reply:
thanks, now https://www.microbialgenomic.cn/gz/gcluster-v2.0.7.tar.gz is available.